src/cls/IPM/Repo/Oras/PackageService.cls

```diff
@@ -194,7 +205,7 @@ Method ListModules(pSearchCriteria As %IPM.Repo.SearchCriteria) As %ListOfObject
 			}
 			#; get all versions
 			Set allTagsString = ..GetAllTagsPy(..Location, package, "", client)
-			Do ..ListModulesFromTagString(allTagsString, tVersionExpression, client, tList, pSearchCriteria, name)
+			Do ..ListModulesFromTagString(allTagsString, tVersionExpression, client, tList, pSearchCriteria, package)
```

Comment:
This change brought to my attention that we check for name being empty but don't handle a case when name isn't provided. We should right since its not required, for example with `-list-modules` where we want to list all modules in a repo?

Reply:
The missing case is no package name in a registry with no namespace. That isn't actually our case for now so we should defer it IMO.